app/boards/shields/helix/README.md

```diff
@@ -0,0 +1,12 @@
+#### Note to user:
+
+* If desired, RGB underglow must be manually enabled before building and flashing. Check 'helix.conf' to do so.
+* Peripheral RGB function is impaired until full support is implemented in the master branch.
+* OLED displays are not currently included in this shield. This will be updated after OLED support is live.
+* 'KANA' and 'EISUU' input is currently utilized under the 'LANG1' and 'LANG2' keycodes respectively.
+
+________
```

Comment:
Sorry @KingCoinless, the formatter we're soon planning to enforce project-wide (#568) just caught this ...

```suggestion
- If desired, RGB underglow must be manually enabled before building and flashing. Check 'helix.conf' to do so.
- Peripheral RGB function is impaired until full support is implemented in the master branch.
- OLED displays are not currently included in this shield. This will be updated after OLED support is live.
- 'KANA' and 'EISUU' input is currently utilized under the 'LANG1' and 'LANG2' keycodes respectively.

---
```

Please accept the change.